### PR TITLE
Collect instance tags during inventory refresh

### PR DESCRIPTION
**Pull request by mansam**

### CODE REVIEW

**Comment by mansam** [comment]
@aufi should we be pulling the tags from the server `tags` property, or the `metadata` property? It seems like either could do the job, but they're distinct from one another.

File: app/models/manageiq/providers/openstack/inventory/persister.rb [hunk 1]
```diff
@@ -21,6 +24,13 @@ def initialize(manager, target = nil, collector = nil)
 
   protected
 
+  # TODO: this reads whole table ContainerLabelTagMapping.all.
+  #   Is this expensive for each targeted refresh?
```
Comment (aufi) [inline app/models/manageiq/providers/openstack/inventory/persister.rb]: Good question, maybe @Ladas or @agrare might know?  Otherwise I fine with merging.

Comment (mansam) [reply app/models/manageiq/providers/openstack/inventory/persister.rb]: That's a comment copied from @Ladas' original code ;)

Comment (aufi) [reply app/models/manageiq/providers/openstack/inventory/persister.rb]: Ah, copied from merged code :-)

**Pull request merged**